dotcom-rendering/playwright/lib/load-page.ts

```diff
@@ -31,6 +31,9 @@ const loadPage = async (
 					`{"value":"${new Date().toISOString()}"}`,
 				);
 			}
+			// force Ophan to use the https endpoint
+			// @ts-expect-error - window type
+			window.ophanRemoteHost = 'https://ophan.theguardian.com';
```

Comment:
Necessary until [this fix](https://github.com/guardian/ophan/pull/5911) is merged